package Exception;

class Demo {
	public int method(int[] arr,int index) {
		System.out.println(arr[index]);//虚拟机进行了对象的封装
		
	}
	
	return arr[index];
}

public class ExceptionDemo2 {

	public static void main(String[] args) {
		
		
		int[] arr = new int[3];
		Demo d = new Demo();
		d.method(arr, 3);
		
//		method(arr,3);
		
	}

	public static void method(int[] arr, int index) {
		
		System.out.println(arr[index]);
		
	}

}
